components/CrawlUrls.tsx

```
@@ -0,0 +1,254 @@
+                    {isPending ? (
+                      <Checkbox
+                        id={`select-${url}`}
+                        checked={selectedUrls.has(url)}
+                        onCheckedChange={(checked) => handleUrlSelectionChange(url, checked)}
+                        disabled={isCrawlingSelected}
+                        aria-label={`Select URL ${url}`}
+                        className="border-white" // Added white border for visibility
+                      />
+                    ) : (
+                      // Render placeholder or nothing if not pending
+                      <div className="h-4 w-4"></div> // Placeholder to maintain alignment
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)